lib/t_web/channels/user_socket.ex

```diff
@@ -58,7 +58,7 @@ defmodule TWeb.UserSocket do
   defp maybe_location(_params), do: nil
 
   defp check_version(nil), do: false
-  defp check_version(version), do: Version.match?(version, ">= 5.0.0")
+  defp check_version(version), do: Version.match?(version, ">= 6.0.0")
```

Comment:
discontinuing support of older versions, the majority of users are on 6+ already